public/pages/Dashboard/Components/AnomaliesLiveChart.tsx

```diff
@@ -137,132 +161,203 @@ export const AnomaliesLiveChart = (props: AnomaliesLiveChartProps) => {
+            All matching detectors are under initialization or stopped for the
+            last 30 minutes. Please adjust filters or come back later.
+          </p>
+        </EuiText>
+      ) : (
+        // show below content as long as there exists anomaly data,
+        // regardless of whether anomaly grade is 0 or larger.
+        [
+          <EuiFlexGroup style={{ marginTop: '0px' }}>
+            <EuiFlexItem>
+              <EuiStat
+                description={'Last updated time'}
+                title={liveTimeRange.endDateTime.format('MM/DD/YYYY hh:mm A')}
+                titleSize="s"
+                style={{ color: '#000' }}
               />
-              <Axis
-                id={'left'}
-                title={'Anomaly grade'}
-                position={Position.Left}
-                domain={{ min: 0, max: 1 }}
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiStat
+                description={'Detector with most recent anomaly occurrence'}
+                title={
+                  lastAnomalyResult === undefined
+                    ? '-'
+                    : get(lastAnomalyResult, AD_DOC_FIELDS.DETECTOR_NAME, '')
+                }
+                titleSize="s"
+                style={{ color: '#000' }}
               />
-              <BarSeries
-                id={'Detectors Anomaly grade'}
-                xScaleType={ScaleType.Time}
-                timeZone="local"
-                yScaleType="linear"
-                xAccessor={AD_DOC_FIELDS.PLOT_TIME}
-                yAccessors={[AD_DOC_FIELDS.ANOMALY_GRADE]}
-                splitSeriesAccessors={[AD_DOC_FIELDS.DETECTOR_NAME]}
-                data={prepareVisualizedAnomalies(visualizedAnomalies)}
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiStat
+                description={'Most recent anomaly grade'}
+                title={
+                  lastAnomalyResult === undefined
+                    ? '-'
+                    : get(lastAnomalyResult, AD_DOC_FIELDS.ANOMALY_GRADE, 0)
+                }
+                titleSize="s"
+                style={{ color: '#000' }}
               />
-            </Chart>,
-          ]
-        )}
-      </div>
+            </EuiFlexItem>
+          </EuiFlexGroup>,
+          <div>
+            {[
+              // only show below message when anomalousDetectorCount >= MAX_LIVE_DETECTORS
+              anomalousDetectorCount >= MAX_LIVE_DETECTORS ? (
+                <EuiCallOut
+                  size="s"
+                  title={`You are viewing ${MAX_LIVE_DETECTORS} detectors with the most recent anomaly occurrences.`}
+                  style={{
+                    width: '88%', // ensure width reaches NOW annotation line
+                    marginTop: '20px',
+                    marginBottom: '20px',
+                  }}
+                >
+                  <p>
+                    10 detectors with the most recent anomalies are shown on the
```

Comment:
Small: remove the hardcoded `10` here to be `${MAX_LIVE_DETECTORS}`

Reply:
Good catch. Thanks.